### ISSUE TITLE
JAR dependencies are not included in the classpath during dexing for tests.

### ISSUE THREAD

**Issue by sgrif** [scala-android/sbt-android#111]

**Comment by sgrif** [comment]
> Details?

JAR dependencies, meaning a JAR file that you have placed in `lib`. In a standard sbt project, the classes provided by that JAR are available in your tests. With this plugin I had to manually add them to `unmanagedClasspath in Test`

Also going to leave my final response to the previous thread here, since you clearly would rather put your fingers in your ears than receive a bug report.

> multidex and scala are slow in general. if you have a suggestion to make it faster, do so.

Try building any project with the gradle toolchain. MultiDex is not slow in general, the way you are accomplishing it is slow. The official android toolchain has incremental multidex, which is much faster.

> In any case, testing works fine, if it doesn't work for you, then I'm sorry. That's all I can offer.

I'm trying to think of some witty retort here, but I'm honestly just speechless at the response of denying that bugs exist. I've noticed that I have a few issues that I never fully responded on. I'll make sure I've left you with the fullest information I have on concrete issues I've had. Your attitude here is completely ridiculous, and makes open source maintainers in general look bad.
